MapboxDirections/MBRouteStep.swift

```diff
@@ -602,6 +643,11 @@ open class RouteStep: NSObject, NSSecureCoding {
             maneuverDirection = nil
         }
         
+        guard let roadSideDescription = decoder.decodeObject(of: NSString.self, forKey: "roadSide") as String?, let roadSide = RoadSide(description: roadSideDescription) else {
+            return nil
+        }
+        self.roadSide = roadSide
```

Comment:
Will this be an issue for old routes that are encoded without a `roadSide`?

Reply:
Yes, let’s assume it’s `right` if unspecified. That would match OSRM’s behavior prior to Project-OSRM/osrm-backend#4415.

/cc @danpat